docs/CodingStandards.md

```diff
@@ -196,6 +198,48 @@ be meaningfully ignored, e.g.:
 LLPC_NODISCARD size_t getFileSize(const std::string& path);
 ```
 
+`Vkgc::Result` can be converted to `std::error_code` using `Llpc::resultToErrorCode`.
+
+### Internal APIs
+
+LLPC tries to follow the LLVM conventions for
+[recoverable error handling](https://llvm.org/docs/ProgrammersManual.html#recoverable-errors).
+In short, functions that either return a value or fail should return `llvm::Expected<T>`,
+while functions that do not return any value should have `llvm::Error` for the return type.
+
+LLPC has its own concrete error type, `Llpc::ResultError`, which bridges
+`llvm::Error`/`Expected<T>` and `Vkgc::Result`. In addition to being compatible with the
+usual LLVM error handling utilities like `llvm::cantFail`, `llvm::handleErrors`, etc.,
+we allow `llvm::Error`s to be converted to `Vkgc::Result` using
+`result = Llpc::reportError(std::move(err))` and `result = Llpc::errorToResult(std::move(err))`.
+See [`llpcError.h`](llpc/util/llpcError.h) for more details.
```

Comment:
This link should point to `../llpc/util/llpcError.h`.

Reply:
Thanks, fixed.